db/data/20160719133546_preload_locids.rb

```diff
@@ -0,0 +1,19 @@
+class PreloadLocids < ActiveRecord::Migration
+  CLASSES = [Axiom, Theorem].freeze
+
+  def self.up
+    LocId.where(specific_type: CLASSES.map(&:to_s)).find_each do |object|
```

Comment:
Here, you could use `Sentence.descendants` instead of `CLASSES`, as well. (more descriptive)